Makefile

```diff
@@ -84,7 +84,13 @@ test-vendor:
 	@hash govendor > /dev/null 2>&1; if [ $$? -ne 0 ]; then \
 		go get -u github.com/kardianos/govendor; \
 	fi
-	govendor status +outside +unused  || exit 1
+	govendor list +unused | tee /tmp/wc-gitea-unused
```

Comment:
The var TMPDIR (or TEMPDIR) seems  to not be set when I run it via `drone exec`. I will add `TMPDIR := $(shell mktemp -d)`